peachjam/templates/peachjam/_judgment_table.html

```diff
@@ -13,8 +13,15 @@
         <tr>
           <td>
             <a href="{{ document.get_absolute_url }}">{{ document.title }}</a>
+            {% include 'peachjam/_labels.html' with labels=document.labels.all %}
+          </td>
+          <td>
+            {% for bench in document.bench.all %}
+              {# djlint:off #}
+              {{ bench.judge.name }}{% if not forloop.last %},{% endif %}
+              {# djlint:on #}
+            {% endfor %}
           </td>
```

Comment:
@longhotsummer I thought we removed these because some judgments have many judges which makes the table look a bit odd. Maybe we could limit the list to 3 or 4?

Reply:
Hmm, fair point. It's useful information for users to have, particularly those who know the judges.

Let's think it through in more detail, because a long list on mobile is particularly bad. Maybe we can do something like the first 3/4 and have "... and 3 more" or something.

Reply:
I agree. We'll need a better way of displaying long lists.

Reply:
I've removed this change based on feedback from the team that the judges aren't actually useful in the listing.